src/components/networked.js

```diff
@@ -411,6 +408,21 @@ AFRAME.registerComponent('networked', {
       // Call networkUpdatePredicate first so that it can update any cached values in the event of a fullSync.
       if (this.networkUpdatePredicates[i](syncedComponentData) || fullSync) {
         componentsData = componentsData || {};
+        if (componentName === 'rotation') {
+          // We compared the rotation, but we are sending actually the quaternion.
```

Comment:
"but we are sending actually" -> "but we actually send" or, "but we are actuallying sending"

Reply:
Thanks, I changed to "but we actually send".